src/models/profile.ts

```diff
@@ -68,7 +69,10 @@ export const Profile = t.intersection([
 
     // array of user's preferred languages in ISO-3166-1-2 format
     // https://it.wikipedia.org/wiki/ISO_3166-2
-    preferredLanguages: PreferredLanguages
+    preferredLanguages: PreferredLanguages,
+
+    // if true this profile is only for test purpose
+    isTestProfile: withDefault(IsTestProfile, false)
```

Comment:
can you check that the generated IsTestProfile has already a default value ?

Reply:
- check on tests in [aa27b03]()

- generated code : `
export type IsTestProfile = t.TypeOf<typeof IsTestProfileBase>;
const IsTestProfileBase = t.boolean;

export const IsTestProfile = withDefault(
  IsTestProfileBase,
  false as IsTestProfile
);
`

Reply:
as it already has a default value we must chagne this to

```suggestion
    isTestProfile: IsTestProfile
```